server/services/interventionsService.ts

```diff
@@ -62,6 +63,11 @@ export interface UpdateDraftEndOfServiceReportParams {
   outcome: CreateEndOfServiceReportOutcome | null
 }
 
+export interface CreateReportDateParams {
+  fromIncludingDate: string
```

Comment:
Can we use `CalendarDay` for these values instead of `string`, and then call `.iso8601` on those inside `getServiceProviderReportingData`? That way callers don't need to know how they're meant to serialize the date values, and it's clear that the from / to are dates, not datetimes.

Reply:
Yep, sounds good